index.js

```diff
@@ -15,10 +12,8 @@ var Key = require('./key');
 var CachingWriter = {};
 
 CachingWriter.init = function(inputTrees, options) {
-  this._super.apply(this, arguments);
```

Comment:
I believe this `_super` call erroneously passed `inputTrees` as the `options` argument to the [base class constructor](https://github.com/ember-cli/core-object/blob/9138b1f55da74386b9547110ee5b6f52afcdfa65/core-object.js#L16), causing a bit of havoc. It's not needed anyway, so I removed it.